### DISCUSSION TITLE
New Groups get added automatically?

### DISCUSSION THREAD

**Discussion by ghost** [trix7777/m2strm#23] [General]
Great software! I know that new groups go to the newgroups.log file but it appears that they also get scrapped. Is there a setting or a way to keep new groups from getting scrapped. I run m2strm once a week so I check the newgroups log and added them to the uwgroups.cfg and decide if I want them or not.

I'm running linux so I could cat the two files together (newgroups and uwgroups) and rerun  if need be but just wanted to check.`

**Comment by trix7777** [comment]
Yes, that is a design feature not to miss out on new groups, I probably had the intention somewhere back to make it an option but right now it's not. And as you say use cat or sed or a combination can make it work the way you want it
